Add a way to choose Podman instead of Docker (#472)
rust-embedded/cross: README.md

accesses it.

### Explicitly choose the container engine

> By default, `cross` tries to use [Docker] or [Podman], with the former to have
> priority. In you want to use other container engine or explicitly choose one, you
> can set the name of binary (or the path) in `CROSS_CONTAINER_ENGINE` environment
> variable.

Reviewer: ```suggestion
By default, `cross` tries to use [Docker] or [Podman], in that order.
If you want to choose a container engine explicitly, you can set the
binary name (or path) using the `CROSS_CONTAINER_ENGINE`
environment variable.
```
